dms/views/directory.xml

```diff
@@ -422,7 +422,7 @@
                         >
                             <field
                                 string="Files"
-                                name="count_total_files"
+                                name="count_files"
```

Comment:
If I'm right, this change will see for example 25 files, although clicking on it will show only the files in the same folder, which can be less. Is it this way? If so, I think it's also confusing, and the change should be to improve the labels on the other stats.

Reply:
It is just the opposite (now it will be consistent).

**Before**:
![antes](https://github.com/OCA/dms/assets/4117568/6921faf9-48a2-44ca-9ea6-963d0d6dc482)

**After**:
![despues](https://github.com/OCA/dms/assets/4117568/c697e45b-f004-49a5-b0ee-010c4c6c30a2)